cmd/kube-proxy/app/server_windows.go

```diff
@@ -185,6 +186,10 @@ func getProxyMode(proxyMode string, kcompat winkernel.KernelCompatTester) string
 	return proxyModeUserspace
 }
 
+func detectNumCPU() int {
+	return goruntime.NumCPU()
```

Comment:
Is this OK?  Does it need followup?

Reply:
Should I put a comment in this line or skip calling getConntrackMax() by judging runtime.GOOS  at https://github.com/kubernetes/kubernetes/blob/d6bc98fc5a39e7ff16dc563408b35767024b13b5/cmd/kube-proxy/app/server.go#L671-L677
In fact no windows will come to this point.